src/build.md

```diff
@@ -71,6 +71,14 @@ A Flix package file (fpkg) is essentially zip-file of the project source code. A
 Flix package, together with its `flix.toml` manifest, can be published on
 GitHub.
 
+### Building a native executable
+
+It is possible to run the GraalVM `native-image` using the
+jar file that to obtain a native executable.
+However, this is not something we officially support so
+your mileage may vary.
+[See the official documentation for native-image for more information.](https://www.graalvm.org/latest/reference-manual/native-image/)
```

Comment:
This text tells me nothing. I was expecting a sequence of what commands to run...

E.g. starting with 

```
$ gradlew clean jar
```

Reply:
and ending with say a comparison of performance:

```java -jar flix.jar Xperf --n 21 --frontend```

vs

``./flix Xperf --n 21 --frontend```

Reply:
roger

Reply:
Oh but this is not for the compiler. It's for a jar built by the flix compiler, i.e., someone writes a `Main.flix`, builds it, builds a jar and then runs native image.
I have done a quick benchmark (using one of the Flix examples) that finds the first 1000 primes and it takes 4.7 seconds using the java jar, and 2.5 seconds with the native executable.
I can write some more detailed instructions for the docs if you'd like.

Reply:
Oh-- then I don't see the point. I  thought it was for the compiler.

Reply:
ah ok

Reply:
Maybe it's nice to know?

Reply:
Well, we cannot guarantee it, because the Flix bytecode format is not yet stable.

Reply:
It would be more interesting if the compiler could be compiled with gradle.